apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -35,9 +205,217 @@ export default class ProjectCard extends React.Component {
       isPublicGallery && isDetailView && projectData.publishedAt;
     const noTimeOnCardStyle = shouldShowPublicDetails ? {} : styles.noTime;
 
+    const {
+      showReportAbuse,
+      isAnyCheckboxSelected,
+      checkboxes,
+      showSubmittedHeader,
+      submitButtonEnabled,
+      reportButtonHovered,
+      reportButtonPressed,
+      showSubmitConfirmation,
+    } = this.state;
+
+    const captchaSiteKey = getScriptData('projects').recaptchaSiteKey;
```

Comment:
I think its a little unusual to be grabbing information directly from a script tag in a React component...I think something like setting it as a page constant when we load the page:

https://github.com/code-dot-org/code-dot-org/blob/b3a27e10f1a86075d898d317476c7c07fb76e8f9/apps/src/javalab/Javalab.js#L196

Then grabbing it from redux, like we do here:

https://github.com/code-dot-org/code-dot-org/blob/27ef1ee6b8eb42296e5de02437f48e2d904f6e6a/apps/src/javalab/JavalabCaptchaDialog.jsx#L77

We should chat through it though, I'm not exactly sure what the easiest approach will be in your case!